packages/ui-kit/src/components/rss-feeds.js

```diff
@@ -9,7 +9,8 @@ import RssFeedTable from './rss-feed-table';
 async function fetcher(...args) {
   const rssParser = new Parser();
   const promises = args.map(async (url) => {
-    const feedData = await rssParser.parseURL(url);
+    const feedString = await (await fetch(url)).text();
+    const feedData = await rssParser.parseString(feedString);
```

Comment:
Hmm, we're still importing the library here which is still not ideal.

I think we should drop this library and use a client-only solution that works in browsers.

For example: https://css-tricks.com/how-to-fetch-and-parse-rss-feeds-in-javascript/

Or find a library that does that but that works in browsers without using nodejs APIs.